ironfish/src/wallet/account.ts

```diff
@@ -29,7 +29,7 @@ export class Account {
   readonly id: string
   readonly displayName: string
   name: string
```

Comment:
This is the main change in the PR that caused cascading of all the other changes.